glitch/analysis/rules.py

```diff
@@ -57,10 +60,14 @@ def to_csv(self) -> str:
 
     def __repr__(self) -> str:
         with open(self.path) as f:
-            line = f.readlines()[self.line - 1].strip() if self.line != -1 else self.repr.split('\n')[0]
-            return \
-                f"{self.path}\nIssue on line {self.line}: {Error.ALL_ERRORS[self.code]}\n" + \
-                    f"{line}\n" 
+            try:
+                line = f.readlines()[self.line - 1].strip() if self.line != -1 else self.repr.split('\n')[0]
+                return \
+                    f"{self.path}\nIssue on line {self.line}: {Error.ALL_ERRORS[self.code]}\n" + \
+                        f"{line}\n"
+            except IndexError:
+                return \
+                    f"{self.path}\nIssue on line {self.line}: {Error.ALL_ERRORS[self.code]}\n"
```

Comment:
Can you explain what is happening here with the try...except?

Reply:
I think this was a fix to an issue that at the time I could not identify but some time later I fixed it while doing other tests and forgot to remove this.